packages/live-preview-sdk/src/index.ts

```diff
@@ -317,3 +317,4 @@ export class ContentfulLivePreview {
 
 export { LIVE_PREVIEW_EDITOR_SOURCE, LIVE_PREVIEW_SDK_SOURCE } from './constants';
 export * from './messages';
+export * from './csm';
```

Comment:
do we export it differently now?

Reply:
I've removed this sorry should have double-checked this! thanks for spotting